examples/react/src/workers/mupdf.worker.ts

```diff
@@ -0,0 +1,63 @@
+/// <reference lib="webworker" />
+import * as Comlink from "comlink";
+import { ColorSpace, Document } from "mupdf";
```

Comment:
We can also use the higher level wrapper ( which is preferable ) 
So:
`import { ColorSpace, Document } from "mupdf";` - > `import { ColorSpace, PDFDocument } from "mupdf/mupdfjs";`